CONTRIBUTING.md

```diff
@@ -26,6 +26,16 @@ npm run watch
 npx playwright install
 ```
 
+**Experimental watch mode for recorder/trace-viewer/UI Mode**
```

Comment:
I think "watch mode" is overloaded with the test runner watch mode. Maybe dev mode, or HMR mode?

```suggestion
**Experimental dev mode for recorder/trace-viewer/UI Mode**
```

Reply:
I would definitely call it out as HMR mode.